version/update.go

```diff
@@ -100,7 +105,7 @@ func readCache(updateFilePath string) *latestInfo {
 	}
 
 	versions := &latestInfo{}
-	err = json.Unmarshal([]byte(contents[1]), versions)
+	err = json.Unmarshal([]byte(strings.Join(contents[1:], " ")), versions)
```

Comment:
please instead use `contents := strings.SplitN(..., 2)` and here use contents[1] which will be the rest of the file.

This will help to avoid Split and Join of the same data.